packages/scripts/scripts/test-playwright.js

```diff
@@ -27,7 +28,10 @@ const {
 
 const result = spawn(
 	'node',
-	[ require.resolve( 'playwright-core/cli' ), 'install' ],
+	[
+		path.resolve( require.resolve( 'playwright-core' ), '..', 'cli.js' ),
+		'install',
+	],
```

Comment:
@swissspidy, @kevin940726, I had to change the CLI path resolution to this (reason #22612).

Reply:
If it works it works :)

Reply:
Nit: Could we just use `npx`?

Reply:
I tried that when I initially found the issue, but it failed locally for me. Pascal might have more details on why we're using `node` vs. `npx`.